src/main.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+import 'source-map-support/register';
```

Comment:
If components are to be imported individually, this probably needs to be in each entrypoint.